doc/lua_api.txt

```diff
@@ -1892,7 +1892,7 @@ Call these functions only at load time!
       * The provided function should check that the position is protected by the mod
         calling this function before it prints a message, if it does, to allow for
         multiple protection mods.
-* `minetest.register_on_item_eat(func(hp_change, replace_with_item, itemstack, user, pointed_thing))`
+* `minetest.register_on_item_eat(func(hunger_change, replace_with_item, itemstack, user, pointed_thing, hp_change))`
```

Comment:
@est31 it does not break the API, see https://github.com/minetest/minetest/pull/3388/files#diff-83bb4564c6fbf230569bdeb2becc32b9R435

When hunger is disabled it falls back to old behavior. You can't expect to add a hunger system and keep the healing by default then. A fallback is enough to not break anything.